c2corg_api/models/enums.py

```diff
@@ -124,3 +124,23 @@ def enum(name, types):
     'article_type', attributes.article_types)
 feed_change_type = enum(
     'feed_change_type', attributes.feed_change_types)
+event_type = enum(
+    'event_type', attributes.event_types)
+author_status = enum(
+    'author_status', attributes.author_status)
+activity_rate = enum(
+    'activity_rate', attributes.activity_rate)
```

Comment:
Please also check the other fields. The enum name is usually singular and the names in `attributes` plural.